### PR TITLE
create CyberHuey.md

### PR DESCRIPTION
**Pull request by CyberHuey**
Fixes #664

### Description

- Added Name, Location, OS, Programming Languages, Personal Info, and picture to personal MDwiki page

### Raw.Githack preview link
raw.githack link to page(s) changed.

https://raw.githack.com/cyberhuey/CyberHuey.github.io/Cyberbranch/#!pages/vi/profiles/CyberHuey.md

### CODE REVIEW

**Comment by dogi** [comment]
@CyberHuey nice start ... it would be nicer if title and description of PR would be more up to code ;)
please fix that

**Comment by CyberHuey** [comment]
What exactly do you mean more up to code. I don't mind making adjustments

**Comment by jayala-29** [comment]
@CyberHuey   

you left the Pull Request description template as it was, you need to replace with what you did 

e.g.  update the description with what you did, take `https://raw.githack.com/cyberhuey/CyberHuey.github.io/Cyberbranch/#!pages/vi/profiles/CyberHuey.md` and put it under the `Raw.Githack preview link` section

**Comment by CyberHuey** [comment]
@dogi @jayala-29 

Please check my latest updates in the descriptions as requested

File: pages/vi/profiles/CyberHuey.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Huey Freeman
+
+- Austin, TX (CST)
+- Windows 10 Pro
+---
+## Programming Langauges
+- C/C++
+- Python
+- HMTL5/CSS
```
Comment (Pavivenkatesan) [inline pages/vi/profiles/CyberHuey.md]: HTML 5... Spell check

Comment (Pavivenkatesan) [reply pages/vi/profiles/CyberHuey.md]: Great to know you Huey !

**Review by Pavivenkatesan — CHANGES_REQUESTED** [review]
See comments

**Review by CyberHuey — COMMENTED** [review]
Spelling corrected!

**Review by law-lin — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Pull request merged**